src/web/app/pages-student/student-course-details-page/student-course-details-page.component.ts

```diff
@@ -100,19 +98,23 @@ export class StudentCourseDetailsPageComponent implements OnInit {
       if (!this.teammateProfiles) {
         this.statusMessageService.showWarningMessage('You do not have any teammates yet.');
       }
+
+      if (this.teammateProfiles && this.course) {
+        const courseId: string = this.course.id;
+        this.teammateProfiles.forEach((teammateProfile: TeammateProfile) => teammateProfile.photoUrl
+          = `${environment.backendUrl}/webapi/student/profilePic?courseid=${courseId}
+          &studentemail=${teammateProfile.email}`);
```

Comment:
The email is not the email in the teammteProfile. It is the email in `StudentAttributes`.

Reply:
Ahh yess! Fixed.